retro/examples/interactive.py

```diff
@@ -245,9 +245,10 @@ def main():
     parser.add_argument('--game', default='Airstriker-Genesis')
     parser.add_argument('--state', default=retro.State.DEFAULT)
     parser.add_argument('--scenario', default=None)
+    parser.add_argument('--record', default=None)
```

Comment:
You should add the ability to pass the argument without a value so it can be set to `True` too:

```suggestion
    parser.add_argument('--record', default=None, nargs='?', const=True)
```

Reply:
Thanks @endrift , I have updated the suggested changes